src/controllers/authenticationController.ts

```diff
@@ -115,6 +122,12 @@ export default class AuthenticationController {
     //
     // decode the SPID assertion into a SPID user
     //
+    const isUserElegibleForIoLoginUrlScheme =
+      getIsUserElegibleForIoLoginUrlScheme(
+        IOLOGIN_USERS_LIST,
+        IOLOGIN_CANARY_USERS_SHA_REGEX,
+        FF_IOLOGIN
+      );
```

Comment:
Instead to redefine this function every call to `acs` endpoint can we define it outside the function, or even outside the class.

Reply:
done in https://github.com/pagopa/io-backend/pull/1012/commits/3ada102f061d73a9a4fc9258f840af9f6d3e6b7b